### DISCUSSION TITLE
How to "partition" data by user into the contol panel?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#9841] [Archive]
Is't possible to configure directus so the user logged see only its data?

**Comment by benhaynes** [comment]
Yes, you can use permissions to limit item access where some `owner` field is set to `$CURRENT_USER`.

https://docs.directus.io/configuration/filter-rules/#dynamic-variables

**Reply by biapar** [reply]
I see the link. This is only if I use a query. How do into the control panel?

**Reply by benhaynes** [reply]
No, this applies to the App too. You can use these in the Role's Permission Builder in the App Settings.
